pkg/oci/builder_test.go

```diff
@@ -44,9 +44,33 @@ func TestBuilder_Build(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	last := path(f.Root, fn.RunDataDir, "builds", "last", "oci")
+	last := filepath.Join(f.Root, fn.RunDataDir, "builds", "last", "oci")
 
-	validateOCIStructure(last, t) // validate it adheres to the basics of the OCI spec
+	validateOCIStructure(last, t) // validate OCI compliant
+}
+
+// TestBuilder_BuildPython ensures that, when given a Python Function, an
+// OCI-compliant directory structure is created on .Build in the expected path.
+func TestBuilder_BuildPython(t *testing.T) {
+	root, done := Mktemp(t)
+	defer done()
+
+	client := fn.New(fn.WithVerbose(true))
+
+	f, err := client.Init(fn.Function{Root: root, Runtime: "python"})
```

Comment:
Looks like the unit test is failing on this

```
--- FAIL: TestBuilder_BuildPython (0.72s)
    builder_test.go:68: exec: "python": executable file not found in $PATH
```

I guess the CI server does need updates?